pkg/utils/params.go

```diff
@@ -19,3 +21,16 @@ func GetParams(line, regEx string) (paramsMap map[string]string) {
 	}
 	return
 }
+
+//GeneratePassword returns string with defined length and random characters
+func GeneratePassword(length int) string {
```

Comment:
We generate redis pass why don't we use the same for sftp pass?

Reply:
I think, it is good point for the future PRs. This PR is about redis password

Reply:
ok